recipe/bld.bat

```diff
@@ -0,0 +1,5 @@
+ls -laF
+
+%PYTHON% %SRC_DIR%\src\sas\qtgui\convertUI.py
+
+python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I'd recommend using `pip` for installation. I think it'd make sense to update the `build.sh` file as well.

Per the [documentation](https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip) + added `--no-deps` so it does not attempts to install the dependencies from PyPI:
```suggestion
%PYTHON% -m pip install . -vv --no-deps
```